### PR TITLE
added flex to ol container and modifired bill item li

### PR DESCRIPTION
**Pull request by JosChavz**
Though it's a little bit more lines of code, I figured you probably need something a little more efficient in your code using flexbox. Though I am not saying the method you used were wrong, it actually worked fine, but I think you should also play around with flexbox properties in case there was actual content underneath the 'receipt-bill' element. 

I just added one character in your HTML file, nothing bad, just caught it.

As for 'receipt-bill', I saw that you wanted to give your children an equal amount of spacing in between and I saw that you used flex on the children but not the parent, which is fine, but you might as well use it on the parent. I gave the parent a fixed height because without it will be automatic from what the children's height is. Give it a larger height and things get interesting. As for 'margin' I wanted to give it a similar look to the original file so I gave it also a bottom margin.

Now that the child is a flex item, the children of it will be smushed together and so to fix that, I added a 'justify-content: space-between'. (This means that line 114, 'bill-price' declaration does not require 'margin-left' property anymore. I didn't remove because I noticed just now and I didn't want to do another commit for it.)

Sweet and clean code nonetheless!

### CODE REVIEW

**Comment by JosChavz** [comment]
Totally forgot how github works. I should have removed the lines!

File: l01s03/styles.css [hunk 1]
```diff
@@ -76,7 +76,13 @@ ol {
 }
 
 .receipt-bill {
-  margin-top: 2.42rem;
+  /*START*/
+  margin: 2.42rem 0 1.7rem 0;
+  display: flex;
+  flex-direction: column;
+  height: 115px;
```
Comment (termosa) [inline l01s03/styles.css]: @JosChavz I wonder why did you add `height`? If new items will be added to the bill they will be squashed. I don't think that it is expected behavior. So I actually put an image in a way that you can add as many items as you want and they will push it accurately without overlaying.

Thank you for your comments and a fix!

Comment (JosChavz) [reply l01s03/styles.css]: @termosa I see what you mean! The method I tried using does limit the amount of children to be added and yours does not, quite neat!

**Comment by termosa** [comment]
I do not merge this PR to keep the bill responsive.

**Comment by termosa** [comment]
I do not merge this PR to keep the bill responsive.
